node/src/chain/acceptor.rs

```diff
@@ -102,6 +102,21 @@ impl<DB: database::DB, VM: vm::VMExecution, N: Network> Acceptor<N, DB, VM> {
         }
     }
 
+    pub fn needs_update(blk: &Block) -> bool {
+        //TODO Remove hardcoded epoch
+        if blk.header().height % 2160 == 0 {
```

Comment:
Maybe we can re-use any  shared constant in Rusk for the epoch value?

Reply:
True, this is why I've TODO, since EPOCH is only inside stake contract right now